x-pack/plugins/endpoint/endpoint_app_types.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface AlertData {
```

Comment:
probably worth marking everything here as readonly in order to prevent @kqualters-elastic's coding style. If you want an easy way you could try: https://github.com/microsoft/TypeScript/issues/13923#issuecomment-557509399